PennMobile/Home/Cells/Group Invites/HomeGroupInvitesCellItem.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  HomeGroupInvitesCellItem.swift
+//  PennMobile
+//
+//  Created by Daniel Salib on 2/7/20.
+//  Copyright © 2020 PennLabs. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+final class HomeGroupInvitesCellItem: HomeCellItem {
+    static var associatedCell: ModularTableViewCell.Type {
+        return HomeGroupInvitesCell.self
+    }
+    
+    var invites: GSRGroupInvites
+    
+    init(invites: GSRGroupInvites) {
+        self.invites = invites
+    }
+    
+    func equals(item: ModularTableViewItem) -> Bool {
+        guard let item = item as? HomeGroupInvitesCellItem else {return false}
+        return invites.count == item.invites.count
+    }
+    
+    static var jsonKey: String {
+        return "invites"
+    }
+    
+    static func getItem(for json: JSON?) -> HomeCellItem? {
+        // comment out following line once server is updated to show cell
+        guard let json = json else { return nil }
+        guard let invites = try? JSONDecoder().decode(GSRGroupInvites.self, from: json.rawData()) else { return nil }
+        return HomeGroupInvitesCellItem(invites: invites)
+    }
+}
+
+extension HomeGroupInvitesCellItem: HomeAPIRequestable {
+    func fetchData(_ completion: @escaping () -> Void) {
+//        GSRGroupNetworkManager.instance.getInvites { (success, invites, error) in
+//            if success {
+//                self.invites = invites
+//            }
+//
+//            completion()
+//        }
+        completion()
+    }
```

Comment:
What data needs to be fetched? Isn't all of it going to provided directly by the `homepage` endpoint? If so, let's get rid of this extension. The only data that we might want to fetch asynchronously are any images associated with the group, but my understanding is that there are no images right now.